deployment/address_book.go

```diff
@@ -33,21 +32,22 @@ var (
 )
 
 type TypeAndVersion struct {
-	Type    ContractType
-	Version semver.Version
-	Labels  LabelSet
+	Type    ContractType   `json:"type"`
+	Version semver.Version `json:"version"`
+	Labels  LabelSet       `json:"labels,omitempty"`
 }
 
 func (tv TypeAndVersion) String() string {
 	if len(tv.Labels) == 0 {
 		return fmt.Sprintf("%s %s", tv.Type, tv.Version.String())
 	}
-	mdSlice := tv.Labels.AsSlice()
-	sort.Strings(mdSlice) // stable order
+
+	// Use the LabelSet's String method for sorted labels
+	sortedLabels := tv.Labels.String()
```

Comment:
nit: Again, isn't "String()" automatically called on types that are passed in as format parameters? I might have it wrong. Anyway, it's not bad to be explicit. /shrug.

Reply:
Ahhh right totally slipped my eyes 😅 will do a smaller cleanup PR tomorrow